docs/content/config/advanced/full-text-search.md

```diff
@@ -137,55 +137,9 @@ While indexing is memory intensive, you can configure the plugin to limit the am
           - ./docker-data/dms/cron/fts_xapian:/etc/cron.d/fts_xapian
     ```
 
-
-### Solr
-
-The [dovecot-solr Plugin](https://wiki2.dovecot.org/Plugins/FTS/Solr) is used in conjunction with [Apache Solr](https://lucene.apache.org/solr/) running in a separate container. This is quite straightforward to setup using the following instructions.
-
-Solr is a mature and fast indexing backend that runs on the JVM. The indexes are relatively compact compared to the size of your total email.
-
-However, Solr also requires a fair bit of RAM. While Solr is [highly tuneable](https://solr.apache.org/guide/7_0/query-settings-in-solrconfig.html), it may require a bit of testing to get it right.
-
-#### Setup
-
-1. `compose.yaml`:
-
-    ```yaml
-      solr:
-        image: lmmdock/dovecot-solr:latest
-        volumes:
-          - ./docker-data/dms/config/dovecot/solr-dovecot:/opt/solr/server/solr/dovecot
-        restart: always
-
-      mailserver:
-        depends_on:
-          - solr
-        image: ghcr.io/docker-mailserver/docker-mailserver:latest
-        ...
-        volumes:
-          ...
-          - ./docker-data/dms/config/dovecot/10-plugin.conf:/etc/dovecot/conf.d/10-plugin.conf:ro
-        ...
-    ```
-
-2. `./docker-data/dms/config/dovecot/10-plugin.conf`:
-
-    ```conf
-    mail_plugins = $mail_plugins fts fts_solr
-
-    plugin {
-      fts = solr
-      fts_autoindex = yes
-      fts_solr = url=http://solr:8983/solr/dovecot/
-    }
-    ```
-
-3. Recreate containers: `docker compose down ; docker compose up -d`
-
-4. Flag all user mailbox FTS indexes as invalid, so they are rescanned on demand when they are next searched: `docker compose exec mailserver doveadm fts rescan -A`
-
 #### Further Discussion
 
 See [#905](https://github.com/docker-mailserver/docker-mailserver/issues/905)
+See [#4024](https://github.com/docker-mailserver/docker-mailserver/issues/4024)
```

Comment:
```suggestion
```

Not sure if this link is useful to the docs page with Solr gone? What was the intent to keep a reference to it?

Reply:
I thought that if someone finds that it stopped working for them and they wonder why it's not working anymore they could refer to that issue to get more information.

Nothing *that* important so if you want to get rid of it I don't have anything against it.

Reply:
> and they wonder why it's not working anymore 

They probably won't easily recognize "See <number link>" as relevant given that the page itself isn't solr specific.

One would hope that they're more likely to check the `CHANGELOG.md` since the breakage shouldn't be a surprise when upgrading to a new major release where we clearly document this sort of thing 👍